packages/shared/src/graphql/search.ts

```diff
@@ -125,7 +125,7 @@ export const SEARCH_HISTORY_QUERY = gql`
       }
       edges {
         node {
-          id: sessionId
+          sessionId: id
```

Comment:
But with this your tracking on the history button will break right?
Used as `target_id: node.sessionId`

Reply:
this is actually the correct way to alias an attribute in a query -- what I had before was actually telling the client to query for the `sessionId` attribute and alias it to `id`, instead of the other way around, which is why the graphql query is failing.

I did not catch this earlier because I could not find an easy way of testing search history locally.

Reply:
Oh I see alias in graphQL are reversed 🤦